targets/TARGET_Freescale/TARGET_KSDK2_MCUS/TARGET_MCU_K64F/device/TOOLCHAIN_GCC_ARM/MK64FN1M0xxx12.ld

```diff
@@ -52,9 +52,11 @@ ENTRY(Reset_Handler)
 
 __ram_vector_table__ = 1;
 
-/* Heap 1/4 of ram and stack 1/8 */
-__stack_size__ = 0x8000;
-__heap_size__ = 0x10000;
+/* Heap 1/4 of ram and stack 1/8. 0x1000 is reduced from both heap
+ * and stack to accomodate static data introduced by GCOV.
+ */
+__stack_size__ = 0x7000;
+__heap_size__ = 0x09000;
```

Comment:
@bogdanm Jimmy introduced a new feature called build profiles. Where you specify all the build macros and flags as an input ```json``` file. Example: https://github.com/ARMmbed/mbed-os/blob/master/tools/profiles/default.json

Coverage can also be enabled by creating a new coverage profile. However, since this PR was implemented before above feature and code coverage can't be enabled on whole source this PR is the way it is now.

With Fastmodels we will not have memory limitations and we can create a build profile and enable coverage on whole source. But this area is under investigation.